besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -207,6 +207,12 @@ protected KeyLoader getKeyLoader() {
   // CLI options defined by user at runtime.
   // Options parsing is done with CLI library Picocli https://picocli.info/
 
+  @Option(
+      names = "--identity",
+      paramLabel = "<String>",
+      description = "Identification for this node in the Client ID")
+  private final Optional<String> identityString = Optional.empty();
```

Comment:
Suggestion: specify `arity` for this option.

Reply:
done

Reply:
Done